src/registry/interfaces/registry-vault.interface.ts

```diff
@@ -1,4 +1,5 @@
 import { VaultState } from '../../api';
+import { VaultRegistryV2Entry } from '../../registry.v2/interfaces';
```

Comment:
can we add this to the registry interfaces export index.ts

Reply:
ofc